tests/unit/DatePicker.spec.ts

```diff
@@ -25,14 +25,16 @@ function createComponent (
   title: string = '',
   errorMsg: string = '',
   disablePicker: boolean = false,
-  inputRules = [(v: string) => v === 'Valid text' || 'Invalid']
+  inputRules = [(v: string) => v === 'Valid text' || 'Invalid'],
+  initialValue: string = ''
```

Comment:
I think this should be `null`, and include a test for this not being set, to verify that existing users of this component don't break.

Reply:
I see `dateText` is initialized to '' in the component, line 91, so the new code shouldn't break anything. (Though, possibly, to avoid multiple initializations, I'd change line line 91 to assign null.)

I'd still like to see a test where `initialValue` is not set, to verify that dateText is initialized to ''.

Reply:
I see `dateText` is initialized to '' in the component, line 91, so the new code shouldn't break anything. Though, possibly, to avoid multiple initializations (*), I'd change line line 91 to assign null.

I'd still like to see a test where `initialValue` is not set, to verify that dateText is initialized to ''.

(*) = multiple initializations is the "decoy pattern": it looks like code is doing something, but it isn't

Reply:
`dateText` is now initialized as `null` on line 91. 

I added the test where `initialValue` is not set and also updated the prop logic as the base set was a bit hard to work with.  I'm just manually providing the props in each test as opposed to using the default `createComponent` function.

Also added a story test specifically for `initialValue` prop.